headers.js

```diff
@@ -2,6 +2,7 @@ require('dotenv').config()
 
 const cspMeta = Object.entries({
   'default-src': ["'self'"],
+  'child-src': ["blob:"],
```

Comment:
- adding an explicit `child-src` directive will override the `'self'` from `default-src`, so that needs to be included too
- we should also do `'report-sample'` to mirror `script-src`, since this directive supports it
```suggestion
  'child-src': ["'self'", "blob:", "'report-sample'"],
```

Reply:
Done in 6ee5814